diff --git a/Android.mk b/Android.mk
index faef5bc..eac09bb 100644
--- a/Android.mk
+++ b/Android.mk
@@ -8,7 +8,7 @@ common_SRC_FILES := \
 	lib/xmlrole.c \
 	lib/xmltok.c
 
-common_CFLAGS := -Wall -Wmissing-prototypes -Wstrict-prototypes -fexceptions -DHAVE_EXPAT_CONFIG_H
+common_CFLAGS := -Wall -Wmissing-prototypes -Wstrict-prototypes -fexceptions -DHAVE_EXPAT_CONFIG_H -DXML_STATIC
 
 common_C_INCLUDES += \
 	$(LOCAL_PATH)/lib
